systems/plants/ContactForceTorqueSensor.m

```diff
@@ -119,7 +119,7 @@
       % contact normal and tangential directions in world coordinates
       nD = length(tsmanip.LCP_cache.data.contact_data.d);
       nC_body = nA + nB;
-      nC = manip.getNumContactPairs;
+      nC = size(tsmanip.LCP_cache.data.contact_data.normal,2);
```

Comment:
What's the reason behind this change? It's clear what getNumContactPairs does, and that it'll always return the same answer--the use of a cache is not as clear.